File: test/fixtures/sample-repo/src/breadth/greeter.rb
module Greeting
  class Greeter
    def greet(name)
      format_name(name)
    end
    def format_name(name)
      name.upcase
    end
  end
end
def run
  Greeting::Greeter.new.greet("x")
end

module Util
  def helper(x)
    x
  end
end

File: test/fixtures/sample-repo/src/breadth/Greeter.swift
struct Point { var x: Int; var y: Int }
class Greeter {
  func greet(_ name: String) -> String { return format(name) }
  func format(_ name: String) -> String { return name.uppercased() }
}
func run() -> String { return Greeter().greet("x") }
